util/release-odemis

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+# Release a new Odemis release
+#
+# Prerequisites are that you can build the Odemis package and upload it to the PPA.
+# That means your computer has the build tools installed (devscripts), configured
+# a signature, and are allowed to upload packages to the odemis PPA (on launchpad.net)
+# The odemis git repo is clone in ~/development/odemis, and the main github repo is
+# called "upstream".
+#
+# Beforehand, make sure you have already:
+#  1. switched to the right branch (eg, v2.6-branch)
+#  2. updated setup.py with the correct version number (eg, 2.6.1)
+#  3. updated the changelog with dch -i (version should be like "2.6.1-1")
+#  4. all is commited
+# Call with "release-odemis".
+#
+# Note: for a new major release (eg v2.7), it does NOT create a new branch
+# Create it (after updating the changelog) with:
+# git checkout -b v2.7-branch $commit-number-to-branch-from$
+#
+# It will:
+#  1. Tag the git repo with the changelog as description
+#  2. Push to the upstream remote
+#  3. Switch to ../pkg-native/odemis/ and update the repo
+#  4. Build for bionic (18.04), focal (20.04) and jammy (22.04).
+
+if [ ! -d ~/development/pkg-native/odemis ]; then
+     echo "Need directory ~/development/pkg-native/odemis with a clone of the odemis repository. Creating one now..."
+     # Do so with:
+     mkdir -p ~/development/pkg-native/
+     cd ~/development/pkg-native/
+     git clone https://github.com/delmic/odemis.git
+fi
+
+cd ~/development/odemis
+
+# Uncomment this line to run line-per-line
+# trap 'read -p "run: $BASH_COMMAND"' DEBUG
+
+# Get the version number from the debian/changelog file
+PVERSION=$(dpkg-parsechangelog -S Version) # should be like "3.1.2-1"
+if ! [[ "$PVERSION" =~ .*-1$ ]]; then
```

Comment:
So far the odemis changelog says that we always have a first successful build, but what if the first build fails on the PPA and we need to re-build locally with a new build number?

Reply:
This script just doesn't handle a rebuild after a build failure. If this happens, this will have to be done manually.